lib/intl.ts

```diff
@@ -170,12 +170,12 @@ function DateTimeFormatImpl(
 }
 
 Object.defineProperty(DateTimeFormatImpl, 'name', {
-  writable: true,
+  writable: false,
```

Comment:
We don't run our polyfill against the whole test262 suite (and that's probably not a good idea), but from browsing it looks like this property is actually intended to be configurable: https://github.com/tc39/test262/blob/main/test/intl402/DateTimeFormat/name.js

Reply:
Oh!  I'll revert.  I assumed this was a mistake. Serves me right for not checking the spec first.